### PR TITLE
[PayumBundle] Decouple payment extension from SyliusCore

### PR DESCRIPTION
**Pull request by ahmadrabie**

### CODE REVIEW

**Comment by ahmadrabie** [comment]
Can you please re-run the build? Seems like false-fail

**Comment by pjedrzejewski** [comment]
@ahmadrabie done! :)

**Comment by aramalipoor** [comment]
:+1:

File: src/Sylius/Bundle/PayumBundle/Extension/UpdatePaymentStateExtension.php [hunk 1]
```diff
@@ -26,7 +23,6 @@ class UpdatePaymentStateExtension implements ExtensionInterface
      * @var FactoryInterface
      */
     protected $factory;
-
```
Comment (Zales0123) [inline src/Sylius/Bundle/PayumBundle/Extension/UpdatePaymentStateExtension.php]: Sadly, these all removed blank lines should be reverted, due to coding standards principles.

**Comment by ahmadrabie** [comment]
@Zales0123 Oops, my bad! It's done ;)

**Comment by michalmarcinkowski** [comment]
Thanks Ahmad!

**Pull request merged**